airbyte-webapp/src/views/Connector/ConnectorForm/components/Sections/PropertySection.tsx

```diff
@@ -47,13 +62,20 @@ export const PropertySection: React.FC<PropertySectionProps> = ({ property, path
 
   const hasError = !!meta.error && meta.touched;
 
-  const errorValues = meta.error === "form.pattern.error" ? { pattern: property.pattern } : undefined;
-  const errorMessage = <FormattedMessage id={meta.error} values={errorValues} />;
+  const errorMessage = Array.isArray(meta.error) ? (
+    <FlexContainer direction="column" gap="none">
+      {meta.error.filter(Boolean).map((error, index) => {
+        return <ErrorMessage key={index} error={error} property={property} />;
+      })}
```

Comment:
I don't think it is very useful to repeat the same error message multiple times, but this is good for showing multiple errors if they are different.

Could we unique the error messages here so there aren't duplicates?

Reply:
Good point, I'm going to adjust this